Makefile

```diff
@@ -7,7 +7,7 @@ quality:
 	go test -race -v -coverprofile cover.out ./...
 
 build:
-	go build -o ${BINARY} .
+	go build -buildvcs=false -o ${BINARY} .
```

Comment:
What is this for?

Reply:
Issue with the Linux build https://github.com/argoproj-labs/argocd-vault-plugin/actions/runs/6526155961/job/17719605595